packages/foreman/foreman/foreman.spec

```
@@ -778,7 +778,9 @@ install -Dp -m0644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/%{name}.conf
 install -Dp -m0644 extras/systemd/%{name}.service %{buildroot}%{_unitdir}/%{name}.service
 install -Dp -m0644 extras/systemd/%{name}.socket %{buildroot}%{_unitdir}/%{name}.socket
 
+%if %{?scl:1}%{!?scl:0}
```

```suggestion
%if 0%{?scl:1}
```

Isn't that more readable?

And yeah, probably also made simpler in "replace shebangs and binaries in scripts for SCL" section :)

actually, why don't we do this step in the replace shebangs part of `%build`? :)

---

> actually, why don't we do this step in the replace shebangs part of `%build`? :)

Good question and feel free to blame me for approving it. I've also wondered the same thing in the past.